.drone.yml

```diff
@@ -0,0 +1,75 @@
+---
+kind: pipeline
+type: kubernetes
+name: devhub-frontend
+
+trigger:
+  branch:
+    # TODO replace with staging
+    - DEVHUB-819
```

Comment:
I'm assuming the TODO will be done post-merge?

Reply:
no. I'm gonna replace this file, with staging and master accordingly. I could not think of better way of testing the push to kanopy. Wish they had three env - dev, staging and prod in kanopy.

Reply:
got it